example/map/src/Main.purs

```diff
@@ -1,22 +1,25 @@
 module Main where
```

Comment:
Similar issue while trying to install deps in `example/map` as described in https://github.com/f-o-a-m/purescript-react-map-gl/pull/5/files#r188905079